src/v2/Apps/Consign/Routes/SubmissionFlow/ContactInformation/ContactInformation.tsx

```diff
@@ -82,7 +82,7 @@ export const ContactInformation: React.FC<ContactInformationProps> = ({
           relayEnvironment,
           submission,
           user?.id,
-          !isLoggedIn ? sd.SESSION_ID : undefined
+          !isLoggedIn ? getENV("SESSION_ID") : undefined
```

Comment:
**question (non-blocking):** Why was this change needed? (Are we deprecating sharify?)